core/store/migrations/migration1568390387/migrate.go

```diff
@@ -1,35 +0,0 @@
-package migration1568390387
```

Comment:
Good point, in this case I removed the old migrations simply because we can, and it avoids unnecessary table creation when we run from scratch.

There will be no impact to any node operator migrating from any version of the database, since we use `DROP TABLE IF EXISTS`.